.github/workflows/auto-labeling.yml

```
@@ -0,0 +1,201 @@
+                color: 'E99695',
+                description: 'Dependency updates'
+              }
+            };
+
+            const changedFiles = '${{ steps.changed-files.outputs.all_changed_files }}';
+            const labelsToAdd = new Set();
+
+            // PRの場合のみファイル変更ベースでラベリング
+            if (context.eventName === 'pull_request' && changedFiles) {
+              const files = changedFiles.split(' ');
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)